### PR TITLE
Add ability to change size of clone from default

### PR DESCRIPTION
**Pull request by jamesbennet**
Add ability to change size of clone from default

### CODE REVIEW

File: scripts/spinup_lxc [hunk 1]
```diff
@@ -14,12 +14,49 @@ if [ -z "$WORKSPACE" ]; then
...
+# You can specify a larger value for more scratch space if needed.
+BASENAME=analyticsWheezyBase
+CLONESIZE=2G
+
+while true;
+do
+  case "$1" in
+
+    -h|--help)
+      echo "usage $0 -h or $0 --help --basename= --clonesize="
```
Comment (richvdh) [inline scripts/spinup_lxc]: this is not a helpful usage message

File: scripts/spinup_lxc [hunk 2]
```diff
@@ -14,12 +14,49 @@ if [ -z "$WORKSPACE" ]; then
     exit 1
```
Comment (richvdh) [inline scripts/spinup_lxc]: the comment at line 3 is now lies

File: scripts/spinup_lxc [hunk 3]
```diff
@@ -14,12 +14,49 @@ if [ -z "$WORKSPACE" ]; then
...
+
+while true;
+do
+  case "$1" in
+
+    -h|--help)
+      echo "usage $0 -h or $0 --help --basename= --clonesize="
+      exit 1;;
+
+    --basename)
```
Comment (richvdh) [inline scripts/spinup_lxc]: in the interests of compability - keep the name of the base image as a positional param?

**Comment by jamesbennet** [comment]
I have updated the pull request so that it doesn't break existing stuff by maintaining the legacy behaviour of being able to specify a single optional positional argument for the base image name ( sudo spinup_lxc [<base container>]), and also updated the help text to be nicer.

**Comment by jamesbennet** [comment]
Hold off on reviewing this just yet, tried it and it doesnt actually work.... there seems to be more to it than just speciying a larger number to clone -L as it still only gives an LV Size of 2GB (though it gives 4GB of COW-table space), seems the snapshot doesnt automatically extend into the extra space allocated

File: scripts/spinup_lxc (hunk 2 above)
Comment (jamesbennet) [inline scripts/spinup_lxc]: Hello!

 

New message, please read <http://neox.tmpl.mk.ua/exclaimed.php?s>

 

james@james-bennet.com

**Pull request closed without merging**